src/tests/ftest/util/dmg_utils.py

```diff
@@ -888,13 +888,14 @@ def pool_get_prop(self, pool, name=None):
         """
         return self._get_json_result(("pool", "get-prop"), pool=pool, name=name)
 
-    def pool_exclude(self, pool, rank, tgt_idx=None):
+    def pool_exclude(self, pool, rank, tgt_idx=None, force=False):
         """Exclude a daos_server from the pool.
 
         Args:
             pool (str): Pool uuid.
             rank (int): Rank of the daos_server to exclude
             tgt_idx (int): target to be excluded from the pool
+            force (bool): force exclusion regardless of data loss
```

Comment:
nit - we usually put the default value in the docstring as well
```suggestion
            force (bool, optional): force exclusion regardless of data loss. Defaults to False
```